sk8prk/header.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * The header for our theme
+ *
+ * This is the template that displays all of the <head> section and everything up until <div id="content">
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ *
+ * @package WordPress
+ * @subpackage Varya
+ * @since 1.0.0
+ */
+?><!doctype html>
+<html <?php language_attributes(); ?>>
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link rel="profile" href="https://gmpg.org/xfn/11" />
+	<?php wp_head(); ?>
+</head>
+
+<body <?php body_class(); ?>>
+<div id="page" class="site">
+	<a class="skip-link screen-reader-text" href="#content"><?php _e( 'Skip to content', 'sk8prk' ); ?></a>
+
+		<header id="masthead" class="site-header wide-max-width">
```

Comment:
It seems like overkill to include this template, considering all that has changed is the class name here to `wide-max-width`.

Reply:
Ahhh, yeah since thats just a responsive class, you can probably add something like this to `_header.scss` and get the same result without this template file (_I think_):

```
.site-header {
    @include %responsive-alignwide-width;
}
```